broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/PartitionTransitionProcess.java

```diff
@@ -84,7 +84,9 @@ private void proceedWithTransition(final ActorFuture<Void> future) {
 
   private void onStepCompletion(final ActorFuture<Void> future, final Throwable error) {
     if (error != null) {
-      LOG.error(error.getMessage(), error);
+      if (!(error instanceof RecoverablePartitionTransitionException)) {
+        LOG.error(error.getMessage(), error);
+      }
```

Comment:
:question: Do you think this is still necessary? I guess we could just remove this as well? Wouldn't it be handled by the `onInstallFailure` above?

Reply:
Yes. You are right. I will remove this.
Should I change the above from warning to error?